meetup/services/meetup_presenter.py

```diff
@@ -0,0 +1,42 @@
+import time
+from pizzaplace.services.pizza_place_presenter import PizzaPlacePresenter
+from pizzaplace.services.yelp_api import YelpApi
+
+class MeetupPresenter():
+  def __init__(self, meetup, meetup_api, api_response_parser):
+    self.meetup = meetup
+    self.parsed_api_response = self.get_api_response(meetup_api, api_response_parser)
+
+  def get_meetup_link(self):
+    return self.meetup.meetup_link
+
+  def get_meetup_name(self):
+    return self.meetup.name
+
+  def get_meetup_venue(self):
+    return self.parsed_api_response.get('venue')
```

Comment:
Here, methods that are talking to a `Meetup` model are asking an object for information about itself, e.g. `self.meetup.name`. Methods using `parsed_api_response` are slicing up a primitive data structure, e.g. `self.parsed_api_response.get('next_event_topic')`. 

Which interface do you find more pleasant to use? Could you use an object to encapsulate the data currently held in `parsed_api_response` and hide the implementation details about how to access its keys?